### ISSUE TITLE
launch fails to build on 10.10.3

### ISSUE THREAD

**Issue by bfontaine** [Homebrew/homebrew#41270]
I installed *all* formulae from the core, this is one of the issues I ran into. I’m at work, will check tonight.

```
$ brew install launch
==> Downloading http://sabi.net/nriley/software/launch-1.2.2.tar.gz
######################################################################## 100,0%
==> xcodebuild -configuration Deployment SYMROOT=build clean
==> xcodebuild -configuration Deployment SYMROOT=build


The following build commands failed:
	Check dependencies
(1 failure)

READ THIS: https://git.io/brew-troubleshooting
```

Logs: https://gist.github.com/8740c69176e777bf580e

**Comment by mralexgray** [comment]
+1

**Comment by mralexgray** [comment]
it should be noted: `brew install launch --HEAD` succeeds.

**Comment by bfontaine** [comment]
Thank you for the info @mralexgray, it means we can fix it with upstream patches (#42201).
